core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -411,8 +415,8 @@ func (eb *EthBroadcaster) handleInProgressEthTx(etx EthTx, attempt EthTxAttempt,
 		//
 		// SCENARIO 3
 		//
-		// The network/eth client can be assumed to have at-least-once delivery
-		// behaviour. It is possible that the eth client could have already
+		// The network/eth Client can be assumed to have at-least-once delivery
```

Comment:
I don't object to this but seems unnecessary?

Reply:
Oh I think this was my refactor tool being overzealous with a rename, fixed